src/components/network-chart/NetworkChartRow.js

```diff
@@ -34,7 +34,12 @@ export type NetworkChartRowBarProps = {
   +marker: TracingMarker,
   // Pass the payload in as well, since our types can't express a TracingMarker with
   // a specific payload.
-  +networkPayload: NetworkPayload,
+  markerMeta: {
+    requestQueue: number,
+    request: number,
+    response: number,
+    type: string,
```

Comment:
The `type` word is a bit overloaded in this project. Normally it is used as a unique string for figuring out what type of thing something is with Flow.

I think it would be better to rename this to `mimeType` or `fileType` to be a bit clearer. I'd prefer `fileType`, as these aren't true mime-types, but the phrase `mime-type` is used elsewhere, so I'd be fine with whatever you think is appropriate here.